support-frontend/assets/components/personalDetails/personalDetailsContainer.tsx

```diff
@@ -64,15 +64,14 @@ export function PersonalDetailsContainer({
 		onEmailChange,
 		onFirstNameChange,
 		onLastNameChange,
+		errors,
 		signOutLink: <Signout isSignedIn={isSignedInUser} />,
 		contributionState: (
-			<ContributionState
-				onChange={(newBillingState) => onBillingStateChange(newBillingState)}
-				selectedState={billingState}
-				isValid={checkBillingState(billingState)}
-				formHasBeenSubmitted={checkoutFormHasBeenSubmitted}
-				contributionType={contributionType}
+			<StateSelect
 				countryGroupId={countryGroupId}
+				state={state}
+				onStateChange={onBillingStateChange}
+				error={errorObject?.state?.[0]}
 			/>
```

Comment:
I believe on the current Contributions checkout `ContributionState` would only render for Single Contributions in the US? Is this the case here?

Reply:
Ah yes sorry, you're correct it only renders on recurring, I knew there was some condition on Contribution type!